infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -166,10 +164,6 @@ public static ReplicationSubscriber getDefaultSubscriber() {
      */
     private void init() {
         registeredSubscribers.add(getDefaultSubscriber());
-        // TODO (V2): This builder should be removed after the rpc stream is added for Sink side session creation.
-        //  and logical group subscribers should come from client registration.
-        registeredSubscribers.add(getDefaultLogicalGroupSubscriber());
-        registeredSubscribers.add(getDefaultRoutingQueueSubscriber());
         registeredSubscribers.add(getDefaultRoutingQueueConfigSubscriber());
```

Comment:
I think we can remove all the default subscribers from "registeredSubscribers".
Since we are going to register the FULL_TABLE as well from within our code, we don't need the DefaultSubscriber as well.